app-web/src/components/UI/AlgoliaBranding/AlgoliaBranding.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Aux from '../../../hoc/auxillary';
+import logo from '../../../assets/images/algolia.svg';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+const noMargin = css`
```

Comment:
using css and style  are probably not necessary at the same time. just move the margin bottom to styled.